lms/envs/common.py

```diff
@@ -1261,17 +1261,15 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'edx-pattern-library/js/modernizr-custom.js',
```

Comment:
Because modernizr-custom is listed here, it is being bundled into the single vendor file (on the courseware, this is lms-main_vendor. However, it is also being downloaded as a separate file. We don't want this situation.

Why did you decide to put it in base vendor? How commonly used is it? The two options are either to remove it from this list and allow it to download as a separate file when needed, or to do special handling in the RequireJS config in LMS so that it does not download separately (this is already done for most everything in this list, see build.js and require-config.js, searching for "underscore" for example).